bin/rpmgrill

```diff
@@ -57,6 +57,7 @@ OPTIONS:
 
   --list-plugins     list available plugins
   --disable=[LIST]   disable one or more plugins, e.g.: --disable=virus,buildlog
+  --blacklist=[FILE] do not report fails in tests listed in blacklist file
```

Comment:
1. `--backlist` seems to be like `--disable`  and I was expecting a list to be passed and is surprised to see file 
how about calling this option `--blacklist-config=[file]` ? 

2. support a config-file option
simple --config-file so that all config related to rpmgrill can be be stored in that file. something like below
```
whilelist:
  - foobar_test
```
NOTE: I used whitelist since it actually **whitelisting** result of a test